core/math/a_star.h

```diff
@@ -169,6 +170,11 @@ class AStar2D : public Reference {
 protected:
 	static void _bind_methods();
 
+	bool _solve(AStar::Point *begin_point, AStar::Point *end_point);
+
+	virtual float _estimate_cost(int p_from_id, int p_to_id);
+	virtual float _compute_cost(int p_from_id, int p_to_id);
```

Comment:
These should be `real_t` since the values come from `distance_to` which uses `real_t`.

Reply:
Fixed...